recipes/syft/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "syft" %}
+{% set version = "0.2.2a2" %}
+
+package:
+  name: {{ syft }}
+  version: {{ 0.2.2a2 }}
+
+source:
+  url: https://pypi.org/project/syft/0.2.0a2/
```

Comment:
You need the full path to the tarball here, not the directory (e.g. https://files.pythonhosted.org/packages/97/ba/c01bb914f395611034bab10cb8d1d22e49691534d30f85efbd82f7bd27cf/syft-0.2.0a2.tar.gz).